aspnet/testing/unit-testing/sample/src/PrimeWeb/Startup.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNet.Builder;
+using Microsoft.AspNet.Http;
+using Microsoft.Framework.DependencyInjection;
+using PrimeWeb.Services;
+
+namespace PrimeWeb
+{
+    public class Startup
+    {
+        // For more information on how to configure your application, visit http://go.microsoft.com/fwlink/?LinkID=398940
+        public void ConfigureServices(IServiceCollection services)
+        {
+        }
+
+        public void Configure(IApplicationBuilder app)
+        {
+            // Add the platform handler to the request pipeline.
+            app.UseIISPlatformHandler();
+            app.UseDeveloperExceptionPage();
```

Comment:
UseDeveloperExceptionPage should only be referenced in a dev environment. See the templates.